wp-content/civi-extensions/goonjcustom/Civi/InstitutionService.php

```diff
@@ -116,7 +138,13 @@ public static function setOfficeDetails(string $op, string $objectName, int $obj
 
     $coordinatorId = $coordinator['contact_id_a'];
 
-    Organization::update(FALSE)
+    Organization::update('Organization', FALSE)
+      ->addValue('Review.Status', 1)
+      ->addValue('Review.Initiated_by', 1)
+      ->addWhere('id', '=', $contactId)
+      ->execute();
+
+    Organization::update('Organization', FALSE)
       ->addValue('Review.Coordinating_POC', $coordinatorId)
       ->addWhere('id', '=', $contactId)
       ->execute();
```

Comment:
why using two diffrent query to update values of single contact of type organization ?

Reply:
I’ve added another query because there’s an issue with the form where hidden default values are being saved. To work around this, I’m updating it through the query.

Reply:
but why you are not updating all together 
```
Organization::update('Organization', FALSE)
      ->addValue('Review.Status', 1)
      ->addValue('Review.Initiated_by', 1)
      ->addWhere('id', '=', $contactId)
      ->execute();
```